decred/decred/util/helpers.py

```diff
@@ -286,124 +135,3 @@ def getLogger(name):
     l.setLevel(LogSettings.moduleLevels.get(name, LogSettings.defaultLevel))
     LogSettings.loggers[name] = l
     return l
-
-
-class ConsoleLogger:
-    """
-    A logger that only prints to stdout.
-    """
-
-    @staticmethod
-    def log(s):
-        print(s)
-
-    debug = log
-    info = log
-    warning = log
-    error = log
-    critical = log
-
-
-def fetchSettingsFile(filepath):
-    """
-    Fetches the JSON settings file, creating an empty JSON object if necessary.
-    """
-    if not os.path.isfile(filepath):
-        with open(filepath, "w+") as file:
-            file.write("{}")
-    return loadJSON(filepath)
-
-
-def saveFile(path, contents, binary=False):
-    """
-    Atomic file save.
-    """
-    with TemporaryDirectory() as tempDir:
-        tmpPath = os.path.join(tempDir, "tmp.tmp")
-        with open(tmpPath, "wb" if binary else "w") as f:
-            f.write(contents)
-            f.flush()
-            os.fsync(f.fileno())
-        shutil.move(tmpPath, path)
-
-
-def appDataDir(appName):
```

Comment:
`appDataDir` may make its way back here eventually. It's based on a go function from **dcrd** that I've seen used in many Decred projects, so may find general use. It's better in **tests** for now though.